packages/framework/src/Resources/translations/messages.cs.po

```diff
@@ -2980,17 +2980,17 @@ msgstr "Přehled prodejů (za posledních 7 dní)"
 msgid "Sales representative"
 msgstr "Obchodní zástupce"
 
-msgid "Sales representative <strong><a href=\"{{ url }}\">{{ name }}</a></strong> was created"
-msgstr "Obchodní zástupce <strong><a href=\"{{ url }}\">{{ name }}</a></strong> byl vytvořen"
+msgid "Sales representative \"%label%\" is assigned to %count% customers and will be removed if you proceed.<br><br> Customers:<br> %customersEnumeration%<br> %extraCount% Do you really want to remove sales representative \"%label%\" permanently?"
+msgstr "[0,20]Obchodní zástupce \"%label%\" je přiřazen k %count% zákazníkům a bude odstraněn, pokud budete pokračovat.<br><br>Zákazníci:<br>%customersEnumeration%<br><br>Opravdu chcete obchodního zástupce \"%label%\" trvale odstranit?|[21,21]Obchodní zástupce \"%label%\" je přiřazen k %count% zákazníkům a bude odstraněn, pokud budete pokračovat.<br><br>Zákazníci:<br>%customersEnumeration%<br>+1 další<br><br>Opravdu chcete obchodního zástupce \"%label%\" trvale odstranit?|[22,Inf]Obchodní zástupce \"%label%\" je přiřazen k %count% zákazníkům a bude odstraněn, pokud budete pokračovat.<br><br>Zákazníci:<br>%customersEnumeration%<br>+%extraCount% dalších<br><br>Opravdu chcete obchodního zástupce \"%label%\" trvale odstranit?"
```

Comment:
let's discuss this please, it does not feel correct

Reply:
why? it seems to be working
lets have a call then